internal/runners/clean/run_lin_mac.go

```diff
@@ -124,11 +119,11 @@ func removeInstall(cfg configurable) error {
 		return errs.Wrap(aggErr, "Could not determine OS specific launcher install path")
 	}
 
-	if err := installation.RemoveSystemFiles(appPath); err != nil {
+	if err := installmgr.RemoveSystemFiles(appPath); err != nil {
 		aggErr = errs.Wrap(aggErr, "Failed to remove system files at %s: %v", appPath, err)
 	}
 
-	installPath, err := installation.InstallPath()
+	installPath, err := installation.InstallPathFromArg0()
```

Comment:
Similar to my other comment, if a user runs `state clean uninstall` how will this get the install path when `os.Args[0]` is an empty string?